lib/render-template.ts

```diff
@@ -37,6 +39,12 @@ function renderTemplate(src: string, dest: string) {
     const existing = JSON.parse(readFileSync(dest, 'utf8'))
     const newPackage = JSON.parse(readFileSync(src, 'utf8'))
     const pkg = sortProperties(deepMerge(existing, newPackage))
+    const cliPackage = JSON.parse(readFileSync('package.json', 'utf8'))
+    const templateName = dirname(src).split(sep).at(-1)
+    if (!['base', 'with-husky'].includes(templateName!)) {
```

Comment:
The deps are sorted at line 41. This should happen before.

Reply:
Why do we exclude with-husky? Moving forward, we will have a new more specialised folder like this one (ex. with-vite)